lib/game_teams_collection.rb

```diff
@@ -331,6 +331,17 @@ def hoa_game_sorter(h_a)
     end
   end
 
+  def head_to_head_hash(teamid)
+    record = Hash.new {|hash, key| hash[key] = []}
```

Comment:
good opportunity to change these variable names from hash and key to be more indicative of what they are working with.